dask/tests/test_distributed.py

```diff
@@ -49,6 +49,11 @@
     ),
 )
 
+mark_ignore_sync_scheduler_warning = pytest.mark.filterwarnings(
+    "ignore:Running on a single-machine scheduler when a distributed client "
+    "is active might lead to unexpected results."
+)
+
```

Comment:
This change is not required for this PR, but I noticed we were repeating the same message several times.